composer.json

```
@@ -20,7 +20,7 @@
     "pimcore/personalization-bundle": "^1.0",
     "pimcore/google-marketing-bundle": "^1.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
-    "symfony/webpack-encore-bundle": "^1.13.2",
+    "symfony/webpack-encore-bundle": "^1.7 || ^2.0",
```

It was probably a typo.

```suggestion
    "symfony/webpack-encore-bundle": "^1.17 || ^2.0",
```